docs/user_roles.md

```diff
@@ -32,13 +32,21 @@ You can do so from the rails console:
 - $ `rails c`
 - > `User.find_by(username: '<your username>').update_attribute(:permissions, 1)`
 
+If you need a higher level of permission above admin. Consider giving your main account super admin permissions instead:
```

Comment:
Grammar fix: "If you need a higher level of permission above admin, consider"

Reply:
Ok, done